extensions/positron-python/src/test/positron/webAppCommands.unit.test.ts

```diff
@@ -180,9 +182,12 @@ suite('Web app commands', () => {
     test('Exec FastAPI in terminal - fastapi-cli not installed, could not infer app name', async () => {
         isFastAPICliInstalled = false;
 
-        sinon.stub(vscode, 'workspace').get(() => ({
+        // TODO @isabelizimm: HACK HACK HACK
+        const mockAsRelativePath = sinon.mock().callsFake(() => ({
             workspaceFolders: [{ uri: { fsPath: '/path/to' } }],
         }));
+        // Replace the original `asRelativePath` method with the mock object
+        vscode.workspace.asRelativePath = mockAsRelativePath;
```

Comment:
I wonder if we can omit this entirely? The other FastAPI tests don't seem to have this stub and if I'm understanding this test correctly, I think the main aspect of this test is passing `undefined` to `verifyRunAppCommand` so the app name cannot be inferred

```suggestion
```